tests/supbot/test_vacancy.py

```diff
@@ -475,6 +533,69 @@ async def test_going_back(
     mocked_bot.assert_no_more_api_calls()
 
 
+@pytest.mark.anyio()
+@pytest.mark.parametrize(
+    ("document_error", "expected_message"),
+    [
+        pytest.param(
+            DocumentErrorType.WRONG_MIME_TYPE,
+            texts.VACANCY_UNSUPPORTED_DOCUMENT_TYPE_MESSAGE,
+            id="wrong_mime_type",
+        ),
+        pytest.param(
+            DocumentErrorType.WRONG_SIZE,
+            texts.VACANCY_UNSUPPORTED_DOCUMENT_SIZE_MESSAGE,
+            id="wrong_size",
+        ),
+    ],
+)
+async def test_handling_unsupported_document(
```

Comment:
suggestion: всё же этот тест чекает именно отправку резюме, а значит должен называться так:
```suggestion
)
async def test_sending_resume_unsupported_document(
```
И быть сразу после `test_sending_resume_unsupported`

Reply:
поправил